package.json

```diff
@@ -12,12 +12,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "git@src.temando.io:developer-experience/temando-open-api-ui.git"
+    "url": "git@github.com:temando/open-api-ui.git"
   },
   "keywords": [
     "ui"
   ],
-  "author": "Florie Guibert",
+  "author": "Temando developers",
```

Comment:
Let's list the contributors for now, https://docs.npmjs.com/files/package.json#people-fields-author-contributors